content/en/examples/pods/qos/qos-pod-4.yaml

```diff
@@ -5,12 +5,12 @@ metadata:
   namespace: qos-example
 spec:
   containers:
-
```

Comment:
I think the added comments make sense. Maybe an existing member can comment for cofirmation.

Reply:
This PR already has an LGTM. With another LGTM I can approve it; I would like a second review from a Kubernetes member before this merges.

Reply:
/assign @sftim 
l thinked this PR be reviewed from other Kubernetes member(@Aut0R3V),
Could you help to add "approved label" ?
Thanks for your help.

/cc @Aut0R3V @rajeshdeshpande02 @steveperry-53
Could you help to revirew this PR ?
Thanks for your help.

Reply:
I’m not sure if we need a comment about the limit provided to the memory since it’s already defined on the manifest though.

Reply:
Otherwise it’s lgtm